extensions/camel/camel-core/deployment/src/main/java/io/quarkus/camel/core/deployment/CamelInitProcessor.java

```diff
@@ -69,7 +69,7 @@ AdditionalBeanBuildItem camelRuntimeProducer(BuildProducer<BeanContainerListener
     }
 
     @Record(ExecutionTime.STATIC_INIT)
-    @BuildStep(applicationArchiveMarkers = CamelSupport.CAMEL_SERVICE_BASE_PATH)
+    @BuildStep(applicationArchiveMarkers = { CamelSupport.CAMEL_SERVICE_BASE_PATH, "org/apache/camel" })
```

Comment:
Minor gripe but should we have `CamelSupport.CAMEL_ROOT_PACKAGE`?

Reply:
I think that would be better, but I don't think it is worth holding up the PR over.